refs/schemas/priv/RFC-PRIV.md

```diff
@@ -476,6 +479,59 @@ It may happen that two or more Retention Policies conflict, e.g. the same data m
 
 In such cases, `NO-LESS-THAN` takes priority over `NO-LONGER-THAN`. The data is kept.
 
+## Lifecycle of Legal Bases for data processing
+
+Each System SHOULD have a knowledge (i.e. a configuration) of a set of [Legal Base](#legal-base) objects.
+Those objects define, for that particular System, what makes it legal to process particular data categories in a particular way for a particular purpose.
+For example, a System might be configured to store and use (`processing-category`:{`STORING`, `USING`}) `CONTACT` data under the `CONTRACT` Legal Base because it needs the data in order to perform a service for the Data Subject.
+The same System might be configured to share (`processing-category`:`SHARING`) `CONTACT` data only under `CONSENT` Legal Base, i.e. if the Data Subject explicitly approves it.
+
+Once the System knows the Legal Bases for each [Privacy Scope](#privacy-scope), it relies on [Legal Base Events](#legal-base-event) and [Consents](#consents) to determine, for each user, the active Legal Bases. E.g. when a Data Subject starts using the System services (`SERVICE-START` type of event) the Privacy Scope associated with `CONTRACT` Legal Base becomes active.
+
+For more details, refer to [Expected Behavior of Implementing Systems](./expected-behavior.md).
+
+### Legal Base
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `legal-base` | 1-* | One or more [Legal Base Term(s)](#legal-bases)|
```

Comment:
`legal-base` should be `1`
`scope` should be `0-*`

Reply:
> `legal-base` should be `1` 

Same idea as [here](https://github.com/blindnet-io/product-management/pull/799#discussion_r921992577). You can create a Legal Base object where you link several legal bases to the same Privacy Scope. The Privacy Compiler resolves them to Privacy Scope Tripples, each associated with one or more Legal Bases.

I think it makes sens for the one making the Legal Base object, to make a lesser number of objects. The burden of resolving its semantics to the atomic level should IMHO be on the Privacy Compiler.

> `scope` should be `0-*`

OK. Changed